src/lib/components/component-collection.ts

```diff
@@ -135,7 +136,8 @@ export const exportedComponents: Array<Type<any>> = [
     FilterComponent,
     TerraInfoComponent,
     TableSettingsComponent,
-    TableSettingsDialogComponent
+    TableSettingsDialogComponent,
+    DeleteFileConfirmationDialogComponent
```

Comment:
Ich glaube es wäre in dem Fall tatsächlich besser einen template dialog zu nutzen. Denn diese Dialog-Component wäre so öffentlich zugänglich. Sollte sie meiner Meinung nach aber nicht sein.

[Hier](https://github.com/plentymarkets/terra-components/pull/1385/files#diff-5728497b7b4e5c58f58bd42d0d465b043ce97096c93dbe1fdfbf17a7392bc5f9) ein Beispiel wie man nen template Dialog nutzt.